src/app/seamly2d/mainwindow.cpp

```diff
@@ -977,20 +977,20 @@ void MainWindow::handleTriangleTool(bool checked)
 
 //---------------------------------------------------------------------------------------------------------------------
 /**
- * @brief handlePointOfIntersectionTool handler for pointOfIntersection tool.
+ * @brief handlePointIntersectXyTool handler for pointOfIntersection tool.
```

Comment:
handlePointIntersectXyTool --- shouldn't this be handlePointIntersectXYTool for consistency?

Reply:
Change made and test build passed.